README.md

```diff
@@ -1,21 +1,17 @@
-gov.au UI-Kit
+GOV.AU UI-Kit
 =============
 
-> The UI-Kit for the Australian Government.
+GOV.AU UI-Kit is a CSS and JS framework for Australian Government websites.
 
+## Contents
 
-## Content
-
-* [Modules](#modules)
+* [Modules & visual tests](#modules)
 * [Build](#build)
-* [Tests](#tests)
 * [License](#license)
 
-
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-
-## Modules
+## Modules & visual tests
```

Comment:
There is a task running that generates this. It now doesn't working anymore because we changed the tag line. Need to talk about this one ;)